src/components/hocs/withStripe.js

```diff
@@ -1,4 +1,6 @@
 import React, { useState, useCallback } from 'react';
+// FEEDBACK: FYI this has been deprecated and replaced by react-stripe-js which provides a more
+//   modern hooks-based approach - defines useStripe() / useElements() hooks, etc.
```

Comment:
I tried that first, but had some problems in the form component using the `stripe` variable, that's why I added this old one. But definitely a nice to have to just use last version